pennylane/devices/__init__.py

```diff
@@ -50,6 +50,7 @@
 
```

Comment:
I can't reach line 45 to comment. There is a typo there: accesssible => accessible